content/academy/apify_platform/getting_started.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Getting started
+description: Get started with the Apify platform by creating an account and learning about the Apify Console, which is where all Apify actors are born!
+menuWeight: 5.1
+paths:
+- apify-platform/getting-started
+---
+
+# [](#getting-started) Getting started
+
+Your gateway into the Apify platform is your Apify account. The great thing about creating an account is that we support integration with both Google and Github, and that it takes only about 30 seconds!
+
+1. Create your account on the [sign up](https://console.apify.com/sign-up) page.
+2. Check your email, you should have a verification email with a link. Click it!
+3. Done! 👍
+
+## [](#getting-to-know-the-platform) Getting to know the platform
+
+Now that you have an account, you have access to the [Apify Console](https://console.apify.com), which is a wonderful place where you utilize all of the features the platform has to offer, as well as manage and test your own projects.
+
+## [](#next) Next up
+
+[Next up]({{@link apify_platform/apify_cli.md}}), you'll be learning about the super cool sidekick to the Apify platform - the Apify CLI. If you've been using `node FILENAME.js` to run your projects, you'll be excited to learn that the Apify CLI allows you to run your Apify SDK projects with some useful flags (such as to purge storage), push and download datasets to and from the cloud, publish your projects to the platform, generate boilerplate code for your new projects, and more!
```

Comment:
I would prefer if we continued the platform explanation now, creating an actor in console a using that to explain all the essential features. We can jump to CLI later.